packages/headless/src/features/commerce/common/actions.test.ts

```diff
@@ -72,7 +71,9 @@ describe('commerce common actions', () => {
       state.cart.cartItems = [product.productId];
       state.cart.cart = {
         [product.productId]: {
-          ...product,
+          [product.price]: {
+            ...product,
+          },
```

Comment:
I think we ended with going for a flat structure with a three part key, rather than nesting price?

Reply:
I thought otherwise, maybe my jet lag clouded my recall :>. As we cater to aggregating on `productId` for CAPI, it is easier for consumers to use the nested prices instead of splitting the key. Also, this makes it possible to add other helpers like `listPrices`.

Reply:
Reverted to flat structure